ssl_cert_tracker/db_helper.py

```diff
@@ -4,53 +4,81 @@
 import time
 import logging
 
-from django.db.models.signals import post_save
-
-from django.core.signals import request_finished
-from django.dispatch import receiver
-
 from django.core.exceptions import ObjectDoesNotExist
-from ssl_cert_tracker.models import TestCertsData, TestCertsHistory
+from ssl_cert_tracker.models import NmapCertsData, NmapHistory
+from django.conf import settings
 
-logging.basicConfig(filename='pocProject.log',level=logging.DEBUG)
+logging.basicConfig(filename='p_soc_auto.log', level=logging.DEBUG)
 
-def Insert_Into_CertsData(json_data):
-    """Insert_Into_CertsData """
-    return_code = "Success"
+def Insert_Into_CertsData(json_data): 
+    """Insert_Into_CertsData check if record already exist then update otherwise insert """
     try:
-        db_certs = TestCertsData()
+        cert_user = NmapCertsData.get_user()

```

Comment:
having so many lines of code inside a try: block is not good practice. please isolate the lines that may raise exceptions into separate try: except: blocks